services/merak-topo/handler/handler.go

```diff
@@ -224,13 +225,19 @@ func Create(k8client *kubernetes.Clientset, topo_id string, aca_num uint32, rack
 		}
 	}
 
-	go Topo_deploy(k8client, aca_image, ovs_image, topo, aca_parameters)
+	namespace = topo_id[:7]
+
```

Comment:
@zzengcs can we use namespace like 'merak-xxxxx` as it's name?

Reply:
addressed